### DISCUSSION TITLE
Opposite of plaintext tag?

### DISCUSSION THREAD

**Discussion by bravecrayon** [maizzle/framework#736] [Q&A] [answered]
If `<plaintext>` can isolate content for plaintext emails, is there an opposite tag or otherwise an approach to isolate content just for the html files? Or a way to use conditionals?

Thanks,
Sean

**Comment by michellaurent** [comment]
It would be nice to have `<not-plaintext>` tag to exclude some elements from the text generated.

**Comment by cossssmin** [comment]
👀

https://github.com/maizzle/framework/pull/765

**Comment by cossssmin** [comment, marked as answer]
Now available in v4.2.0:

https://github.com/maizzle/framework/releases/tag/v4.2.0

**Reply by michellaurent** [reply]
Amazing ! Thanks.

**Reply by bravecrayon** [reply]
Thanks!
